src/main/kotlin/org/rust/lang/template/RustFileContextType.kt

```diff
@@ -6,7 +6,7 @@ import com.intellij.psi.util.PsiUtilCore
 import org.rust.lang.RustLanguage
 
 
-class RustFileContextType : TemplateContextType("RUST_FILE", "Rust file") {
```

Comment:
You can still do that. We should have a single `RustFileContextType` class, and then multiple other context classes that use `RustFileContextType` as their parent context. (see third parameter of the `TemplateContextType` class)